docs/account/createTransaction.md

```diff
@@ -19,26 +19,38 @@ Returns : [Transaction](https://dashevo.github.io/DashJS/#/usage/dashcorelib-pri
 Notes: This transaction will be need to be signed [`account.sign(transaction)`](/account/sign) and then, if wanted, broadcasted to the network for execution `account.broadcastTransaction()`.
 
 Example : 
+```js
+const recipients = [{recipient:"yereyozxENB9jbhqpbg1coE5c39ExqLSaG", satoshis:10e8},{recipient: "yMN2w8NiwcmY3zvJLeeBxpaExFV1aN23pg", satoshis: 1e8}];
+const change = "yaVrJ5dgELFkYwv6AydDyGPAJQ5kTJXyAN";
+const tx = account.createTransaction({recipients, change});
+```
+
+**Strategy:**
+
+By default, wallet-lib is shipped with two different strategies : 
+
+- **simpleDescendingStrategy** : Will try to use as much small inputs as possible up to meeting the necessary amount required.  
+    Allow fee to be optimized for the smallest size at the cost of breaking big inputs in smaller size.
+- **simpleAscendingStrategy** : Will try to use as much small inputs as possible up to meeting the necessary amount required.  
+    Allow to get rid of many small inputs at the cost of potential bigger fee.
+
+You can also pass your own strategy (as a function) to allow you to create your own strategy for how you will want to spend the UTXO.   
+
 ```js
 const recipient = "yereyozxENB9jbhqpbg1coE5c39ExqLSaG";
 const satoshis = 10e8;
 const specialStrategy = (utxosList, outputsList, deductFee = false, feeCategory = 'normal')=> { 
 //...
 };
 const txOpts1 = {
+     recipient, satoshis,
+     strategy: 'simpleAscensingStrategy',
```

Comment:
```suggestion
     strategy: 'simpleAscendingStrategy',
```

Reply:
Wow ! Thank you so much ! Eagle eyes on this *.* <3